graphicsmagick.yaml

```diff
@@ -58,6 +76,8 @@ subpackages:
       - uses: split/dev
 
 update:
+  enabled: false
```

Comment:
Since this is tracked on release monitor, we can keep this set to `true` so that update PRs still come through for review. `manual: true` will flag them so that others know the commit needs to be bumped

Reply:
Thank you so much, RJ, I just fixed it